geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/ExportCommand.scala

```diff
@@ -56,6 +56,9 @@ class ExportCommand(parent: JCommander) extends CommandWithCatalog(parent) with
       exporter.flush()
       exporter.close()
     }
+
+    //necessary to avoid StatsWriter exception when exporting
+    Thread.sleep(1000)
```

Comment:
Yeah, if we don't sleep, zookeeper will log an interrupted error that we can't catch. I'll update the comment.